lib/provider.js

```diff
@@ -1,7 +1,7 @@
 'use babel';
 
 const STEP_DEFINITION_GLOB_CONFIG_KEY = 'cucumber-autocomplete-multipath.stepDefinitionGlob';
-const CUCUMBER_STEP_DEF_PATTERN = /(Given|And|When|Then)\(\/\^(.*?)\$/g;
+const CUCUMBER_STEP_DEF_PATTERN = /(Given|And|When|Then)\((?:'(.*?)'|\/\^(.*?)\$)/g;
```

Comment:
This regex is becoming hard to read now :). I'd like to see some comments or some tests so that it is clear what it is matching